### ISSUE TITLE
No delete functionality for job posting?

### ISSUE THREAD

**Issue by fairbairn** [rizafahmi/elixirjobs#35]
So editing an existing job posting doesn't work.  It thinks it's a New Job every time and doesn't carry over a good bit of the content, including the Job Description.

We also cannot find any way to remove a job posting.

Where is it?

**Comment by rizafahmi** [comment]
Hi @fairbairn , @manusajith already working on this issue. Hi @manusajith it seems delete button is not working right now. Edit description is working. Thanks!

**Comment by optikfluffel** [comment]
@rizafahmi I just tried to delete a job I created and it seems to work fine in Firefox and Chrome, but not in Safari.

**Comment by fairbairn** [comment]
We never saw a delete button at all.  Has one been added?

We were in Firefox Linux.

**Comment by optikfluffel** [comment]
Yes, in https://github.com/rizafahmi/elixirjobs/commit/b85cf136b508385a6056044f9f782ed9605a1de6 by @manusajith.

**Comment by manusajith** [comment]
@rizafahmi @optikfluffel I drilled down the issue to dimmer.js component. It uses `opacity` as a parameter in one of the functions, which won't work in `strict` mode, and was causing js errors and not triggering the DELETE request.

I have edited the parameter of the function directly for now, but I believe a better solution would be a fix in the upstream itself(if not already reported and fixed).

But for the time being it should work :)
